jets/e/blake.c

```diff
@@ -0,0 +1,131 @@
+/* j/5/blake.c
+**
+*/
+#include "all.h"
+
+#include <argon2.h>
+#include "../src/blake2/blake2.h"
```

Comment:
Where is this header coming from? Should the PR include a new submodule?

Reply:
Ah, and that module is already added in the crypto-additions branch. Thanks!

Reply:
NP !